doc/man/man8/daos.8

```diff
@@ -1,295 +1,976 @@
-.br
-	  \fB--dfs-prefix=\fRSTR      <path> a prefix to be trimmed from the absolute path when looking up the dfs-path in the container
-.br
-	  \fB--src=\fRSTR             src=daos://<pool/cont>[/<path>] | <path>
-.br
-	  \fB--dst=\fRSTR             dst=daos://<pool/cont>[/<path>] | <path>
-.br
-		type is daos, only specified if pool/cont used
-.TP
-.I object \fR(\fIobj\fR) \fBCOMMAND\fRs:
-	  \fBquery\fR            query an object's layout
-.br
-	  \fBlist-keys\fR        list an object's keys
-.br
-	  \fBdump\fR             dump an object's contents
-.TP
-.I object \fR(\fIobj\fR) \fBOPTION\fRs:
-	  <\fIpool\fR options>   (\fB--pool\fR, \fB--sys-name\fR)
-.br
-	  <\fIcont\fR options>   (\fB--cont\fR)
-.br
-	  \fB--oid=\fRHI.LO        object ID
-
-.SH COLOPHON
-This page is part of the Distributed Asynchronous Object Storage (DAOS)
-exascale storage project.
-.br
-The project git reposity
-can be found at:
-\%https://github.com/daos-stack/daos.git .
-.br
-Bugs can be reported at:
-\%https://jira.hpdd.intel.com/projects/DAOS/.
+daos is a tool that can be used to manage/query pool content,
+create/query/manage/destroy a container inside a pool, copy data
+between a POSIX container and a POSIX filesystem, clone a DAOS container,
+or query/manage an object inside a container.
+.SH OPTIONS
+.SS Application Options
+.TP
+\fB\fB\-\-debug\fR\fP
+enable debug output
+.TP
+\fB\fB\-\-verbose\fR\fP
+enable verbose output (when applicable)
+.TP
+\fB\fB\-\-json\fR\fP
+enable JSON output
+.SS Help Options
+.TP
+\fB\fB\-h\fR, \fB\-\-help\fR\fP
+Show this help message
+.SH COMMANDS
+.SS container
+perform tasks related to DAOS containers
+
+\fBAliases\fP: cont
+
+.SS Help Options
+.TP
+\fB\fB\-h\fR, \fB\-\-help\fR\fP
+Show this help message
+.SS container check
+check objects' consistency in a container
+
+\fBUsage\fP: container check [check-OPTIONS]
+.TP
+.SS check objects' consistency in a container
+.TP
+\fB\fB\-G\fR, \fB\-\-sys-name\fR\fP
+DAOS system name
+.TP
+\fB\fB\-p\fR, \fB\-\-pool\fR\fP
+pool UUID (deprecated; use positional arg)
+.TP
+\fB\fB\-d\fR, \fB\-\-path\fR\fP
+unified namespace path
+.TP
+\fB\fB\-c\fR, \fB\-\-cont\fR\fP
+container UUID (deprecated; use positional arg)
+.TP
+\fB\fB\-e\fR, \fB\-\-epc\fR\fP
+container epoch
+.SS Help Options
+.TP
+\fB\fB\-h\fR, \fB\-\-help\fR\fP
+Show this help message
+.SS container clone
+clone a container
+
+\fBUsage\fP: container clone [clone-OPTIONS]
+.TP
+.SS clone a container
+.TP
+\fB\fB\-S\fR, \fB\-\-src\fR (\fIrequired\fR)\fP
+source container
+.TP
+\fB\fB\-D\fR, \fB\-\-dst\fR (\fIrequired\fR)\fP
+destination container
+.SS Help Options
+.TP
+\fB\fB\-h\fR, \fB\-\-help\fR\fP
+Show this help message
+.SS container create
+create a container
+
+\fBUsage\fP: container create [create-OPTIONS]
+.TP
+.SS create a container
+.TP
+\fB\fB\-G\fR, \fB\-\-sys-name\fR\fP
+DAOS system name
+.TP
+\fB\fB\-p\fR, \fB\-\-pool\fR\fP
+pool UUID (deprecated; use positional arg)
```

Comment:
Is this a UUID or a UUID or label?

Reply:
Either, technically, once the PR adding label support lands. I expect that the --pool and --cont flags will mostly be used for compatibility with existing scripts/fingers.